addpage.html

```diff
@@ -0,0 +1,123 @@
+            </a>
+            <a href="#">
+                <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5"
+                    stroke="currentColor">
+                    <path stroke-linecap="round" stroke-linejoin="round"
+                        d="M14.857 17.082a23.848 23.848 0 005.454-1.31A8.967 8.967 0 0118 9.75v-.7V9A6 6 0 006 9v.75a8.967 8.967 0 01-2.312 6.022c1.733.64 3.56 1.085 5.455 1.31m5.714 0a24.255 24.255 0 01-5.714 0m5.714 0a3 3 0 11-5.714 0" />
+                </svg>
+            </a>
+        </nav>
+        <div class="addContainer">
+            <div class="addHeader">
+                <a href="index.html">
+                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor">
+                        <path fill-rule="evenodd"
+                            d="M7.72 12.53a.75.75 0 010-1.06l7.5-7.5a.75.75 0 111.06 1.06L9.31 12l6.97 6.97a.75.75 0 11-1.06 1.06l-7.5-7.5z"
+                            clip-rule="evenodd" />
+                    </svg>
+                </a>
+                <h2>Add new people</h2>
+            </div>
+            <div class="inputSection">
+                <form action="" id="allInputs">
+                    <div class="uploadPhoto">
+                        <div class="photoDisplay">
+                            <img id="photoDisplayed"
+                                src="https://i.fbcd.co/products/resized/resized-750-500/563d0201e4359c2e890569e254ea14790eb370b71d08b6de5052511cc0352313.jpg"
+                                alt="a photo">
+                        </div>
+                        <input type="file" id="uploadPhotoBox" style="display:none;"
+                            accept="image/jpeg, image/png, image/jpg">
+                        <label for="uploadPhotoBox">Upload Photo</label>
+                    </div>
+                    <div class="yourName">
+                        <div class="firstName">
+                            <p>First Name</p>
+                            <input type="text" id="yourName" placeholder="e.g: Ram">
+                        </div>
+                        <div class="lastName">
+                            <p>Last Name</p>
+                            <input type="text" id="yourName" placeholder="e.g: Phago">
+                        </div>
+                    </div>
+                    <div class="ourStory">
+                        <p>How did we meet?</p>
+                        <textarea placeholder="share our story..."></textarea>
+                    </div>
+                    <div class="knowEachOther">
+                        <p>Do we know eachother from outside?</p>
+                        <input type="radio" id="knowOutside">
+                        <label for="knowOutside">Yes</label>
+                        <input type="radio" name="No" id="dontKnowOutside">
+                        <label for="dontKnowOutside">No</label>
+                    </div>
```

Comment:
There is no name attribute on Yes , hence I am able to check both the radios which might not be intended here